### DISCUSSION TITLE
dev2 + M5stack core2 AWS

### DISCUSSION THREAD

**Discussion by Szetya** [dl9rdz/rdz_ttgo_sonde#458] [Developers corner]
dev2 + M5stack core2 AWS + M5stack LoRa V1.1 + SKU:U032 GPS

Hi Hansi! 
The configuration in the title works perfectly. Currently it shows me a fingerprint of 22.
It may be slightly different with the AXP configuration (vibration motor, speaker_enable...).
Can you confirm for me if the AXP192 DIO/(A)LDO pins are configured correctly for correct operation?
Since the touch screen is not handled by the code so I assigned the touch interrupt GPIO to button1 so that currently touching any point on the screen makes the button1 event available. 😉

![Screenshot_20240906_055616_com android chrome](https://github.com/user-attachments/assets/b518213e-5867-48af-8454-5295b1b17a19)

**Comment by Szetya** [comment]
The PIN configuration is available here: https://docs.m5stack.com/en/core/core2_for_aws

**Reply by dl9rdz** [reply]
If I see it correctly then the core2 for aws is basically the same as the plain core2 (except for some additional components), so yes this should work and be configured correctly.

The information in the web interfaces needs an update so it shows that it was detected :)

**Comment by Szetya** [comment]
Thanks for the confirmation!
Of course, everything started on the first try. I just had to set the NSS jumper switch in the LoRa unit correctly. GPS was also found. The first time I tried it without GPS the RX/TX PIN autoconfiguration was -1/-1. Second time with GPS connected the RX PIN was in place (TX remained -1).
